components/resolved_ts/src/endpoint.rs

```diff
@@ -371,7 +371,7 @@ where
                         observe_id,
                         cause: format!("met error while handle scan task {:?}", e),
                     })
-                    .unwrap();
+                    .unwrap_or_else(|schedule_err| warn!("schedule re-register task failed"; "err" => ?schedule_err, "re-register cause" => ?e));
```

Comment:
How can it fail?

Reply:
The store is shutting down, the worker's receiver may drop before the task send